internal/criocli/criocli.go

```diff
@@ -128,13 +128,22 @@ func mergeConfig(config *libconfig.Config, ctx *cli.Context) error {
 		runtimes := StringSliceTrySplit(ctx, "runtimes")
 		for _, r := range runtimes {
 			fields := strings.Split(r, ":")
-			if len(fields) != 3 {
+
+			runtimeType := "oci"
```

Comment:
Please make this a global const. Or do we already have this somewhere defined?

Reply:
@saschagrunert, I've addressed this by using `libconfig.DefaultRuntimeType` instead of "oci".

Let me know if this is what you had in mind.